src/main/java/net/dries007/tfc/ConfigTFC.java

```diff
@@ -958,6 +958,10 @@ public static void onConfigChangedEvent(ConfigChangedEvent.OnConfigChangedEvent
             @Config.LangKey("config." + MOD_ID + ".client.tooltip.propickOutputToActionBar")
             public boolean propickOutputToActionBar = true;
 
+            @Config.Comment("Tooltips with any of the following categories will output to the actionbar (the space just above the hotbar)")
+            @Config.LangKey("config." + MOD_ID + ".client.tooltip.tooltipCategoriesOutputToActionBar")
+            public String[] tooltipCategoriesOutputToActionBar = new String[] { "anvil", "vessel", "propick" };
```

Comment:
This is redundant when we already have config values for each of these. Just use those rather than have to invent parsing. (Never mind the fact this is a completely opaque config option that @Claycorp would yell at me for since there's no indication to the user *what* values are valid here. Just use the existing booleans.

Reply:
Originally I thought it would be good to avoid having possibly many options for this and reduce it to a single config value, but you are right that it is very obscure